pyproject.toml

```diff
@@ -51,6 +51,7 @@ loguru = "^0.5.3"
 saxpy = "^1.0.1-dev167"
 hydra-slayer = "^0.2.0"
 typer = "^0.4.0"
+omegaconf = "^2.1.1"
```

Comment:
there is smth wrong with the diff 🤔 
omegaconf is already in master and there is no conflicts in pr

Reply:
I moved it up. So it won't mixed with optional deps.